src/AppBundle/GraphQL/Mutation/CharacterMutation.php

```diff
@@ -36,7 +39,8 @@ function createCharacter(string $userId, string $characterName)
         $this->game->getEntityManager()->flush();
 
         return [
-            "character" => new CharacterType($this->getGame(), $character)
+            "characterEdge" => CharacterConnection::createEdgeFor($user, new CharacterType($this->getGame(), $character)),
+            "user" => new UserType($this->getGame(), $user),
```

Comment:
I have not tried to not return the user - I've been looking around for RANGE_ADD examples and they all return the parent field (user) of the connection + the connection edge in the mutation payload.